controller/jobcontroller.go

```diff
@@ -562,15 +562,7 @@ func handleRmdata(job *apiv1.Job, restClient *rest.RESTClient, clientset *kubern
 	labels := job.GetObjectMeta().GetLabels()
 	clusterName := labels[config.LABEL_PG_CLUSTER]
 
-	//delete any pgtask for this cluster
-	log.Debugf("deleting pgtask for rmdata job name is %s", job.ObjectMeta.Name)
-	err = kubeapi.Deletepgtasks(restClient, config.LABEL_PG_CLUSTER+"="+clusterName, namespace)
-	if err != nil {
-		return err
-	}
-
-	err = kubeapi.DeleteJob(clientset, job.Name, namespace)
-	if err != nil {
+	if err = kubeapi.DeleteJob(clientset, job.Name, namespace); err != nil {
```

Comment:
Unless you care about the error outside of this scope, do `:=` to assign it here. Which would allow you to get rid of declaring it above.

Reply:
Agree with the comment, looking through the code, this would take further work and testing given the meaning of `err` appears to be overloaded further down in this block. Likely a change/refactor we need to come back to later.